integration/kubernetes/init.sh

```diff
@@ -77,7 +77,9 @@ if ip a show "$cni_interface"; then
 fi
 
 echo "Start ${cri_runtime} service"
-sudo systemctl start ${cri_runtime}
+sudo systemctl daemon-reload
+sudo systemctl cat "${cri_runtime}.service"
```

Comment:
debugging ?

Reply:
Yes, I am still figuring out some things while looking at the CI results. Will mark this as WIP till then.